Tribler/Policies/BoostingManager.py

```diff
@@ -0,0 +1,1114 @@
+# -*- coding: utf-8 -*-
+# Written by Egbert Bouman, Mihai Capotă, Elric Milon, and Ardhi Putra Pratama H
+# pylint: disable=too-few-public-methods, too-many-instance-attributes
```

Comment:
I think these lines should be removed. It's probably not good to specify custom pylint behaviour (we have a nice `.pylintrc` file for that).